testrunner-k8s-role.yml.template

```diff
@@ -23,6 +23,10 @@ rules:
   - apiGroups: [""]
     resources: ["pods", "pods/exec"]
     verbs: ["get", "list", "watch", "delete", "patch", "create", "update"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    resourceNames: ["terra-component-version"]
+    verbs: ["get", "patch", "update"]
```

Comment:
This section is adding to the permissions of the Kubernetes SA -- is that correct?

Reply:
That's correct.